test/node/uuid_tests.js

```diff
@@ -162,4 +164,22 @@ describe('UUID', () => {
     const uuid = new UUID(UPPERCASE_DASH_SEPARATED_UUID_STRING);
     expect(inspect(uuid)).to.equal(`new UUID("${LOWERCASE_DASH_SEPARATED_UUID_STRING}")`);
   });
+
+  it(`should serialize as a valid UUID _bsontype with Object input without error`, () => {
```

Comment:
Per discussion, let's add tests to ensure that the serialization result is equivalent to the output of uuid.toBinary

Reply:
Sounds good, just added them and reorganized the serialization tests to be in their own describe block!